### ISSUE TITLE
Two R packages (clusterProfiler and enrichplot)

### ISSUE THREAD

**Issue by shinysyk** [ubccr/software-layer#34]
Hi. My name is Yungki Park. I teach BCH519. Please install clusterProfiler and enrichplot for R 4.1.0. These are widely used for bioinformatics analysis, including analysis of RNA-seq, ChIP-seq, and CUT&Tag data. They are part of Bioconductor. I tried to install these myself in my home directory for the class. But it did not work because the installation process needed, but could not find, some system library files. Thanks for your help!

clusterProfiler: https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html
enrichplot: https://bioconductor.org/packages/release/bioc/html/enrichplot.html

**Comment by aebruno** [comment]
@shinysyk These R packages are available in the `r-bundle-bioconductor` module. For example:

```
$ module load foss
$ module load r-bundle-bioconductor
$ R

R version 4.2.0 (2022-04-22) -- "Vigorous Calisthenics"
Copyright (C) 2022 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

> library('clusterProfiler')

clusterProfiler v4.4.4  For help: https://yulab-smu.top/biomedical-knowledge-mining-book/
```

For more info on using the new software modules. See the [docs](https://docs.ccr.buffalo.edu/en/latest/software/modules/)

**Comment by shinysyk** [comment]
Hi Andrew,

Thanks for your reply. I tried your method. But I got the following error
message:
Lmod has detected the following error: The following module(s) are unknown:
"foss"
Lmod has detected the following error: The following module(s) are unknown:
"r-bundle-bioconductor"

Would you mind looking into this? Thanks for your help.

Best,
Yungki

**Comment by shinysyk** [comment]
Another point to note is that R version 4.2.0 is not available to me. I
guess it is available only to you. Since other R packages for the class are
targeted to R version 4.1.0, if you would not mind, I would like those two
packages to be installed and available for R 4.1.0.

Best,
Yungki

**Comment by aebruno** [comment]
Please confirm you a using the new software layer. Please [read the documentation here](https://docs.ccr.buffalo.edu/en/latest/software/modules/), specifically the text in the big yellow box.

**Comment by aebruno** [comment]
> Since other R packages for the class are targeted to R version 4.1.0, if you would not mind, I would like those two packages to be installed and available for R 4.1.0.

We currently support R version 4.2.0 in the new software layer. We will not be supporting other versions of R at this time.

**Comment by shinysyk** [comment]
Hi Andrew,

Thanks a lot for your help! The issue is now resolved.

Best,
Yungki
